lib/reline/config.rb

```diff
@@ -182,7 +182,7 @@ def read_lines(lines, file = nil)
       next if if_stack.any? { |_no, skip| skip }
 
       case line
-      when /^set +([^ ]+) +([^ ]+)/i
+      when /^set +([^ ]+) +(.+)/i
         var, value = $1.downcase, $2
         bind_variable(var, value)
```

Comment:
For an example input `set foo bar baz`, how about passing three values to bind_variable?
```ruby
var = 'foo'
value = 'bar'
raw_value = 'bar baz'
bind_variable(var, value, raw_value)
```
Most variable still uses value, and some variables (that uses `retrieve_string(value)` now) will use `retrieve_string(raw_value)`

Reply:
Thanks! Added `raw_value` as a third argument to `bind_variable()`